appConfig.js

```diff
@@ -102,7 +102,7 @@ const config = {
   pageTitle: {
     ya: 'Best Books for Teens | NYPL',
     childrens: 'Best Books for Kids | NYPL',
-    'staff-picks': 'Staff Picks | NYPL',
+    'staff-picks': 'NYPL | Staff Picks',
```

Comment:
I don't think this change is necessary. Normally, we use the specific title and then pipe in NYPL, {title} | NYPL for all page titles.

Reply:
Hmmm. Yeah, also a question for Courtney but the content itself doesn't matter much for this PR.

Reply:
The title is from the answer of Courtney, so it should be correct. Since I have updated it then I will just keep it here.